dawn.json

```diff
@@ -537,7 +540,14 @@
                 "returns": "blend state builder"
             },
             {
-                "name": "create buffer builder",
+                "name": "create buffer",
+                "returns": "buffer",
+                "args": [
+                    {"name": "descriptor", "type": "buffer descriptor", "annotation": "const*"}
```

Comment:
I'm probably forgetting something, but for objects this small (two params), could we just pass them as args instead of in a descriptor? Or is it for future expansion?

Reply:
Like you guessed the descriptor gives us extensibility.